storage/mysql/log_storage_test.go

```diff
@@ -967,6 +967,52 @@ func TestGetActiveLogIDsEmpty(t *testing.T) {
 	}
 }
 
+func TestGetUnsequencedCounts(t *testing.T) {
+	numLogs := 4
+	cleanTestDB(DB)
+	logIDs := make([]int64, 0, numLogs)
+	for i := 0; i < numLogs; i++ {
+		logIDs = append(logIDs, createLogForTests(DB))
+	}
+	s := NewLogStorage(DB, nil)
+
+	ctx := context.Background()
+	expectedCount := int64(0)
+
+	for i := int64(1); i < 10; i++ {
+		// Put some leaves in the queue of each of the logs
```

Comment:
All logs have the same number of leaves throughout the iterations. That would hide a bug where we apply the count of a certain log to all others. I'd rather do a single pass, but having different numbers of leaves for each log (possibly with an empty log too).